system-int-test/test/cmr/system_int_test/search/autocomplete/suggestion_reindex_test.clj

```diff
@@ -15,19 +15,24 @@
    [cmr.system-int-test.utils.ingest-util :as ingest]
    [cmr.system-int-test.utils.search-util :as search]))
 
+(defn- scores-descending?
+  [results]
+  (->> results
+       (map :score)
+       (apply >=)))
 
 (defn compare-autocomplete-results
   "Compare expected to actual response for the following:
   - Items are ordered by score in descending order
   - Ensure that the other fields match"
   [actual expected]
-  (let [actual-scores-descending? (->> actual
-                                       (map :score)
-                                       (apply >=))
-        ;; we don't need to actually compare scores
-        expected (map #(dissoc % :score) expected)
+  ;; check score returns
+  (when (>= 1 (count actual))
```

Comment:
this is kinda weird, does (seq actual) work?

Reply:
Hadn't thought to try seq, I'll see if I can switch that over.